src/d3a/models/area/__init__.py

```diff
@@ -103,6 +109,23 @@ def __init__(self, name: str = None, children: List["Area"] = None,
         self.redis_ext_conn = RedisMarketExternalConnection(self) \
             if external_connection_available is True else None
 
+    def validate_baseline_energy_settings(self):
```

Comment:
Could be moved to `d3a-interface` to be used in `d3a-web` too

Reply:
Yes, I was thinking about it but first needed your opinion about the whole calculation and added it here. Will move to d3a-interface and use it in the d3a-web part too.